app/dashboard/templates/bounty/details2.html

```diff
@@ -36,9 +36,6 @@
             Admin Actions
           </span>
           <a v-for="link in quickLinks" class="btn btn-outline-gc-blue px-2 mx-2 font-smaller-4" :href="[[link.href]]" :title="[[ link.title ]]">[[ link.label ]]</a>
-          <button class="btn btn-outline-gc-blue px-2 mx-2 font-smaller-4" @click="contactFunder()">Contact Funder</button>
```

Comment:
these removed are not part of quickLinks since are not links, are prompts and I fixed them to not send the actions if are empty (today that is broken in normal bounties if you click that prompts then the action is sent even without nothing

Reply:
Added this back !